project/src/main/java/com/google/sps/servlets/LogInServlet.java

```diff
@@ -77,6 +83,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
         Gson gson = new Gson();
         response.setContentType("application/json;");
-        response.getWriter().println(gson.toJson(user));
+        if(request.getParameter("logout") != null){
+            response.getWriter().println(gson.toJson(null));
+        }else{
+            response.getWriter().println(gson.toJson(user));
+        }
```

Comment:
Fix if else for spaces please in all files.

Reply:
done.